src/plugins/observables.js

```diff
@@ -13,7 +13,12 @@ export default {
 
     // We know `rule` is a `StyleRule`, and the other types don't have a
     // `prop` method, so we must explicitly cast to `StyleRule`
-    const rule = ((createRule(name, {}, options): any): StyleRule)
+    const rule = ((
+      // `link: true` will fail if `decl` is an empty dict, so we initialize it
+      // with `position: static`.  Since that's the default anyway, this
+      // shouldn't affect actual styling, but gives JSS something to bite into.
+      createRule(name, {position: 'static'}, options)
```

Comment:
nono please revert) I have a better idea, will add a real quick fix, give me a sec

Reply:
it fixes it but we can avoid it completely

Reply:
Great.  I didn't like this solution anyway.  😄